### ISSUE TITLE
Android: Put the application on the app store

### ISSUE THREAD

**Issue by noseshimself** [anywherelan/awl#92]
... for those who can't or do not want to side-load stuff.

**Comment by pymq** [comment]
Hi @noseshimself, thanks for asking

Do you have any specific app store in mind?

There are already two issues about google play store and F-Droid: #39  #31

**Comment by noseshimself** [comment]
The Google Play Store.

I did not give sufficient detail, sorry. Background:

I'm using ChromeOS. Due to the security model (I guess they took a lot of hints from Qubes, lately even more -- ANYTHING they do not have under control is forbidden in the ChromeOS host itself and if you damage their security model (e. g. by adding random Android packages which you may do) going back to full "secure end user mode" requires you to completely wipe the device. Actually a good idea if every piece of Android software (or Linux software) you can install inside virtual machines was able and willing to export their configuration. Linux containers can be backed up. Android... not. 

One major detail of this security model is "you either load Android apps from the Play Store or you enter developer mode and permit side-loading via ADB but if you enable that the Play Store will not consider this a validated device". Of course; the Android (formerly a container within ChromeOS itself) has certain rights that permit it to bypass some ChromeOS security.

As long as your android Application is correctly declaring all required abilities in its manifest ChromeOS will accept it as valid VPN solution (just like everything else from OpenVPN to Wireshark). Which reminds me to ask for a feature: Exportable configuration file(s), just in case. I fyou encrypt them well you can even put them permanently on Google Drive (or, my favorite, NextCloud).

**Comment by pymq** [comment]
Thanks for this detailed background! I didn't know that ChromeOS is such a restricted system.

> to ask for a feature: Exportable configuration file(s), just in case

Yes, I too thought about how it could be hard to move to a new device, so I have already added this feature, it's available from GUI

About publishing in stores, I'm willing to do this, but not until awl reaches critical user base mass. I don't have google's developer account, it costs money (not much, but still), also publishing in google play is not that easy, too. For me it's not worth it, yet, unfortunately

**Comment by pymq** [comment]
Closing as a duplicate of https://github.com/anywherelan/awl/issues/39
